src/services/AccessTokenRequestProcessor.ts

```diff
@@ -96,6 +104,7 @@ export class AccessTokenRequestProcessor {
     			}),
     		};
     	} catch (err: any) {
+    		this.logger.error({ message: "Error while trying to create access token ", err });
```

Comment:
separate key {error: err}

Reply:
I have changed the key to be `error` but not sure what you mean about having two objects, one for message and one for error? Is there anywhere that we are doing that so I can have a look?